src/App/app.css

```diff
@@ -0,0 +1,10 @@
+.app-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 30px;
```

Comment:
Is this a good opportunity to showcase using one of the CSS variables? If it doesn't require pulling in separate babel plugins, might be good to use one just as an example. Otherwise, if it requires configuration changes we could wait and add that later. Just a thought.